src/org/loklak/harvester/TwitterScraper.java

```diff
@@ -416,9 +416,10 @@ public String toString() {
     
     public static class TwitterTweet extends MessageEntry implements Runnable {
 
-        public final Semaphore ready;
-        public UserEntry user;
-        public boolean writeToIndex, writeToBackend;
+        private final Semaphore ready;
+        private UserEntry user;
+        public boolean writeToIndex;
+        public boolean writeToBackend;

```

Comment:
@kavithaenair @SKrPl Adding of getter methods only for unit-tests doesn't seem to be right. So I haven't dropped changes here. what do you say?